eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml

```diff
@@ -3,7 +3,7 @@ steps:
     - pwsh: |
         $filesInCommonDir = git diff "origin/$(System.PullRequest.TargetBranch)" HEAD --name-only -- 'eng/common/*'
 
-        if ((-not $PrSourceBranch.StartsWith("sync-eng/common")) -and ($filesInCommonDir.Count -gt 0))
+        if ((!"$(System.PullRequest.SourceBranch)".StartsWith("sync-eng/common")) -and ($LASTEXITCODE -eq 0) -and ($filesInCommonDir.Count -gt 0))
```

Comment:
Will $LASTEXITCODE ever be hit? Wouldn't this script halt in devops if there's a non-zero exit code?

Reply:
`git diff` should set a lastexitcode but you are correct that maybe the error preferences devops uses might cause it to fail at that point. We'd have to test to know for sure but in either case I don't think it hurts to have the check.